.github/workflows/release.yml

```diff
@@ -55,12 +58,31 @@ jobs:
         run: |
           git pull
           npm install
+      - name: Semantic Release runs for draft release
+        if: ${{github.event.inputs.draft-release == 'true'}}
+        run: npx semantic-release -t \${version}-draft
+        env:
+          GITHUB_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
Acc to sematic-release [tags](https://semantic-release.gitbook.io/semantic-release/usage/configuration#tagformat) could be used to differentiate releases, that's what was the intention here. I like the idea of having a different npmjs for draft release altogether, that would also mean not using the existing workflow. Let me know if you want me to explore this?

Reply:
Yes, I have a GH action for deleting tagged releases, but I am not very confident about it, for the same reasons why `twilio-cli` cannot be unpublished from npm. That's why I wanted to trigger once tomorrow, to verify these. 
Having a private npmjs for these makes more sense now, with that we can even trigger it a couple of days before the actual release to have some buffer time too.

Reply:
This will be solved as a part of [DII-207](https://issues.corp.twilio.com/browse/DII-207), we'd separate the draft tagged release from current public npmjs.